app/views/journeys/_city_journey_card.html.erb

```diff
@@ -28,13 +34,15 @@
       }
     } do |f|
   %>
-    <div class="m-3">
-      <div class="edit-form">
+    <div class="cj-form-container">
+      <div id="edit-cj-date">
         <%= f.input :start_date, as: :string, input_html: { data: { flatpickr_target: "startDate" } } %>
         <%= f.input :end_date, as: :string, input_html: { data: { flatpickr_target: "endDate" } } %>
       </div>
-      <%= f.association :city, collection: City.order("name") %>
-      <%= f.submit "Update journey", class: "btn standard-button mt-2", id:"show-card-button" %>
+      <div id="city-drpdwn">
+        <%= f.association :city, collection: City.order("name") %>
+      </div>
+      <%= f.submit "Update journey", class: "btn update-cj-button" %>
```

Comment:
placed into DIV so that it could be styled specifically, and be styled over the standard simple-form styling (with an ID)